fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -194,6 +197,15 @@ static Key persistTagMessagesKey(UID id, Tag tag, Version version) {
 	return wr.toValue();
 }
 
+static Key persistStorageTeamMessagesKey(UID id, StorageTeamID storageTeamId, Version version) {
```

Comment:
Nit: instead of using `static`, we could wrap these functions inside an anonymous namespace.

Reply:
will DM you, I am not sure what is the best approach for it.